Update main readme and examples readme (#192)
grammarly/grammarly-for-developers: readme.md

# Grammarly for Developers

> [Grammarly for Developers]((https://developer.grammarly.com)) is Grammarly's platform that allows you to add Grammarly's real-time writing support to your apps. This repo is your go-to source for finding [code examples](./examples), reporting [issues](https://github.com/grammarly/grammarly-for-developers/issues), and chatting in the [Grammarly for Developers community](https://github.com/grammarly/grammarly-for-developers/discussions).

Reviewer: ```suggestion
[Grammarly for Developers]((https://developer.grammarly.com)) is Grammarly's platform that allows you to add Grammarly's real-time writing support to your apps. This repo is your go-to source for finding [code examples](./examples), reporting [issues](https://github.com/grammarly/grammarly-for-developers/issues), and chatting with the [Grammarly for Developers community](https://github.com/grammarly/grammarly-for-developers/discussions).
```

[![Grammarly for Developers](https://github.com/grammarly/grammarly-for-developers/blob/main/docs/sdk_product_animation.gif)](https://developer.grammarly.com)

## Code examples 🧑‍💻

This repo contains numerous code examples you can use as you learn about the Grammarly Text Editor SDK. You can find all of the examples in the [examples](./examples) directory. Check out the [examples readme](./examples/readme.md) for a summary of the examples.

## Report an issue 🐞

> If you've run into an issue or found a bug, you can report it by navigating to the [Issues](https://github.com/grammarly/grammarly-for-developers/issues) tab. Issues you report here will be publicly available so other members of the community can benefit from the discussion.

Reviewer: ```suggestion
If you've run into an issue or found a bug, you can report it by navigating to the [Issues](https://github.com/grammarly/grammarly-for-developers/issues) tab. Issues you report here are publicly available so that other members of the community can benefit from the discussion.
```
